### DISCUSSION TITLE
Potatoes and what other food item are the two main ingredients in "bangers and mash"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#400] [Galaxy Trivia] [answered]
**Hint:** 🌭

**Comment by peterbaccaro** [comment]
**Sausages** is the other main ingredient along with potatoes in bangers and mash.  Sausages ended up with the nickname "bangers" during World War I because fillers (mostly water) was used as a filler in the meat due to food shortages and the high water content and tight casings often caused the sausages to explode during cooking.

**Comment by Roberto-Larivera** [comment]
Bangers and mash (Salsicce con purè di patate)

**Comment by thuongtruong1009** [comment, marked as answer]
The other food item that is a main ingredient in "bangers and mash" is sausages. Bangers are a type of sausage, typically made with pork and/or beef, and are often served with mashed potatoes and gravy in the traditional British dish known as bangers and mash.
